spec/controllers/teachers_controller_spec.rb

```diff
@@ -183,4 +183,24 @@
       expect(Teacher.find_by(email: "valid_example@valid_example.edu").not_reviewed?).to be true
     end
   end
+
+  context "flash a message after resend_welcome_email" do
+    it "succeeds when teacher is validated, sets success" do
+      ApplicationController.any_instance.stub(:current_user).and_return(Teacher.find_by(first_name: "Validated"))
+      validated_teacher = Teacher.find_by(first_name: "Validated")
```

Comment:
```suggestion
      validated_teacher = Teacher.find_by(first_name: "Validated")
      ApplicationController.any_instance.stub(:current_user).and_return(validated_teacher)
```

Arguably, this spec should use a factory (or fixture at the very least...) rather than a `find_by` call, but we should perform the call first.